src/azul/deployment.py

```diff
@@ -480,10 +480,11 @@ def _log_client_response(self,
                              **kwargs: Any
                              ) -> Optional[AWSResponse]:
         event_name = self._shorten_event_name(event_name, self._response_event_name)
-        response = kwargs['response_dict']
-        boto3_log.info('%s:\tGot %s response', event_name, response['status_code'])
-        message = http_body_log_message(boto3_log, 'response', response.get('body'))
-        boto3_log.info('%s:\t%s', event_name, message)
+        response = kwargs.get('response_dict')
+        if response is not None:
```

Comment:
I changed my mind on this. 

|  | response is None | response is not None |
|---|---|---|
| **exception is None** | log | log |
| **exception is not None** | don't log | fail |